### DISCUSSION TITLE
Is there any way to get solids to not collide against each other?

### DISCUSSION THREAD

**Discussion by Kentyahoo** [replit/kaboom#695] [Q&A]
I want to get the enemies spawning and moving toward the player with the solid comp but when the enemies collide they get stuck to each other and can't move. So I want the enemies to not collide with each other.  Can you please show me how I do that. Thank you.

My source is here https://github.com/Kentyahoo/MonsterAvenge/blob/main/code/main.js

But here is the logic code:
```js
const fireblob_speed = 100

const fireblobSpec = (x, y) => [
  sprite("fireblob"),
  pos(x, y),
  scale(1.4),
  state("move", ["move"]),
  area(),
  layer("ui"),
  solid(90),
]

onCollide("fireblobs", "fire blobs", () =>{
  pushOutAll()
})

const coords = Array.from({ length: 10 }).map((_, i) => {
  return { x: i*100, y: i + 100}
})
const fireblobs = coords.map(c => {
  const fireblob = add(fireblobSpec(c.x, c.y))
  
  fireblob.onStateUpdate("move", () => {
  	if (!player.exists()) return
  	const dir = player.pos.sub(fireblob.pos).unit()
  	fireblob.move(dir.scale(fireblob_speed))
  })
  return fireblob
})
```

**Comment by lajbel** [comment]
I see you using `2000.2.9` in your repo, I recommend you update to latest beta `3000.0.0-beta.3`, cause there is an option in `area({ collisionIgnore: ["tag1", "tag2"] })`, there is some other breaking changes in v3000, should read [this blog](https://3000.kaboomjs.com/blog/3000). Notice there's an important bug with stretch and letterbox screen resize.

Beyond that, there is no way to ignore collisions according to a tags in version 2000, there's the `layers()` system where every `layer()` ignore collisions with other layers, maybe that can help you.

**Comment by Kentyahoo** [comment]
I cannot find the place where you can update to the kaboom.js version 3000 beta

**Reply by lajbel** [reply]
> We're releasing Kaboom v3000 beta! This major version update includes a lot of new features and improvements, and also breaking changes (see migration guide below).
> 
> To try the beta version, install kaboom@next
> 
> $ npm install kaboom@next
> <script src="https://unpkg.com/kaboom@next/dist/kaboom.js"></script>
> or use [this Replit template](https://replit.com/@slmjkdbtl/kaboom3000)

In the [blog](https://3000.kaboomjs.com/blog/3000) previously linked.

**Reply by Kentyahoo** [reply]
I am having trouble getting it to work.

I installed npm and did `npm install kaboom@next`

I created a HTML file with the following:
```html
<html>
<script src="https://unpkg.com/kaboom@next/dist/kaboom.js"></script>
<script>console.log('hi')</script>    
</html>
```

I see "hi" in console, but the page is blank. 

I don't understand how to "use the Replit template".

Please advise.

thank you.

**Reply by lajbel** [reply]
I think it depends on the setup of your dev environment. I see in your repo you are using NPM, only run inside your project folder `npm i kaboom@next` and you should be ready.
